eeps/eep-0012.md

```diff
@@ -75,7 +75,7 @@ there are four possible outcomes:
 4. Expr yields true => the test succeeds;
    this might as well have been Expr without Pattern,
    and Pattern = true beforehand.
-======================================================================
+
```

Comment:
(I can't comment on lines outside the diff.) A blank line is missing before the numbered list above. The list is currently broken. (See https://erlang.org/eeps/eep-0012.html)

There are more blank lines missing in this EEP, for example before and after every code block in the next section.